### PR TITLE
fix: flap transition bounces in SU4

### PR DESCRIPTION
**Pull request by donstim**
Fixes #[issue_no]

## Summary of Changes

1. Changes to flap system to minimize the vertical bounce associated with transitioning between flap configurations when using SU4. The transition between CONF 1 and CONF2 and between CONF3 and CONF 4 were most affected by this issue.
2. Minor change to vertical tail effectiveness to change Cnβ (weathervaning stability parameter) back to the value used for https://github.com/flybywiresim/a32nx/pull/4666. Something in SU4 changed this value as well as other stability parameters not addressed in this pr. It is also unknown at this point if rudder effectiveness for crosswind control was affected by SU4.
## Additional context

Other than the slight change to the vertical tail effectiveness for weathervaning, there are no changes in this pr for flaps up flight characteristics. All flap/slat configurations has to be redone to minimize any vertical bounce, while meeting the same standards as the previous flight model.

Discord username (if different from GitHub): donbikes#4084

## Testing instructions

1. During a flight, proceed through flap transitions to verify that any vertical bounce has been minimized, and no undesired flight characteristics occur.
2. Part of the solution to minimizing the vertical bounce involved a slight increase in flap deployment time. Ensure flap deployment times remain reasonable. (No data available to objectively check against at this time.)

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg [hunk 1]
```diff
@@ -220,10 +222,10 @@ lift_coef_pitch_rate = -57.116 ; The change in lift per change in pitch rate
 lift_coef_daoa = 0 ; lift per change in angle of attack rate
 lift_coef_delta_elevator = -1.652 ; The change in lift per change in elevator deflection
 lift_coef_horizontal_incidence = 0 ; The change in lift per change in horizontal incidence angle
-lift_coef_flaps = 1.9 ; Change in lift due to flaps
+lift_coef_flaps = 1.715 ; Change in lift due to flaps
 lift_coef_spoilers = -0.25 ; Change in lift due to spoilers
 drag_coef_zero_lift = 0.0197 ; The zero lift drag polar
-drag_coef_flaps = 0.1128
```
Comment (MisterChocker) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg:226]: are these changes intentional?

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg [hunk 2]
```diff
@@ -220,10 +222,10 @@ lift_coef_pitch_rate = -57.116 ; The change in lift per change in pitch rate
...
+drag_coef_flaps = 0.0778
```
Comment (MisterChocker) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg:228]: are these changes intentional?

Comment (donstim) [reply flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg:228]: yes, of course

Comment (donstim) [reply flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg:228]: To give a little more context, those values are essentially the full flaps/slats lift and drag coefficients. (Notice the lift/drag factors for full flaps/slats are 1.00). Any time flaps/slats have to be redone, those values will change. The other configurations then have factors applied to these increments to determine their lift/drag coefficients.

**Comment by aguther** [comment]
I did a test and the bounce is gone. LGTM

**Comment by aguther** [comment]
I suggest adding the force to the modern flight model with this PR

**Review by aguther — APPROVED** [review]

**Comment by donstim** [comment]
Let's do it in a different or separate PR. I haven't checked it, don't have time to, and would like to get this pr through QA with the minimum amount of bother.

**Comment by DavidEGTE** [comment]
Quality Assurance Trainee Report

Discord: David#2177
Object of testing: 4909
Tier of Testing: 1
Date: 28/05/2021

Testing Process: Full flight from LFPG - LROP
Negatives: N/A
Testing Results: Passed

Conclusions: LGTM

**Comment by St54Kevin** [comment]
**Quality Assurance Tester**

Object of testing: #4909 
Tier of Testing: 1
Date: 28/05/2021

**Testing Process:**
- First tested flap bounce on current dev without AP
- Tested the same scenarios with this PR

Negatives: N/A
**Testing Results:** Passed

Conclusions: The flap bounce actually IS reduced in this PR. My experience in the CEOs is, that there is actually a quite significant balooning. I ask our Neo pilots here:[https://discord.com/channels/738864299392630914/754851312306487326/847756629419360276](url) wether what experience they had.

**Comment by sidnov** [comment]
QA Report

Object of testing: #4909
Tier of Testing: 1
Date: 28/05/2021

Testing Process: Standard test previously suggested by donbikes for this particular issue. Spawned at 3000ft, stable flight. Flap 1->Flap 2 around 185kts. Flap 3->Flap Full around F speed. 

Negatives: None
Testing Results: Passed

Conclusions: Previously observed 400-500ft/min ballooning with SU4 and dev is now mitigated to about 200ft/min.
